src/RegistryItemOptionsResolver.php

```diff
@@ -23,5 +27,27 @@ public function __construct(array $options)
 				break;
 		}
 
+		if (null !== $options['metadata']) {
+			foreach (preg_split('/,?\s+/', $options['metadata']) as $name) {
```

Comment:
everything is optional in your regex: https://eval.in/477552

I could do `/,\s*|,?\s+/`, but then opted for a space separated with optional comma. Who delimits using a comma w/o a spaca anyway? :)